### ISSUE TITLE
_cached_content is being overwritten as a function

### ISSUE THREAD

**Issue by dandroid88** [joelverhagen/flask-rauth#7]
```
access_token = resp.content.get('access_token')
```

AttributeError: 'function' object has no attribute 'get'

Commented out the content() override in RauthResponse and everything works.  I am using python 2.7 and OAuth2.  Took me a while to wire up pycharm and confirm.  Just as I step into the if, it changes from a member variable to an instance function...not sure why though.

For now I will forgoe the caching...

**Comment by dantaylor688** [comment]
Wow, thanks. I had the same error. Your comment saved me a bunch of time!

**Comment by dandroid88** [comment]
Absolutely, we forked it, and removed the caching: https://github.com/flair-systems/flask-rauth
Since the forked version wasn't in pip we added this to our requirements.txt:
-e git://github.com/flair-systems/flask-rauth.git@face89b682684addb0637e534664aecc1ff72d24#egg=Flask_Rauth-master
cheers
